### ISSUE TITLE
Sunset this repo

### ISSUE THREAD

**Issue by fmvilas** [asyncapi/design-system#71]
So, as of today, we now have the design system in the Studio repo: https://github.com/asyncapi/studio/tree/master/apps/design-system. Therefore, I'm proposing we close this repo to avoid redundancies. What do you say? @mcturco @magicmatatjahu

**Comment by mcturco** [comment]
Yeah I think its a good call as long as we migrate this issue somewhere where it makes sense to live: https://github.com/asyncapi/studio/issues/662

I guess Studio would make sense for that one too?

All other issues can be closed

**Comment by fmvilas** [comment]
Transferred to Studio. Thanks for pointing it out 👍
